src/avalanchemq/client/client.cr

```diff
@@ -271,11 +267,9 @@ module AvalancheMQ
       true
     rescue ex : IO::Error | OpenSSL::SSL::Error | AMQ::Protocol::Error::FrameEncode
       @log.debug { "Lost connection, while sending (#{ex.inspect})" }
```

Comment:
Will `AMQ::Protocol::Error::FrameEncode` raise something on `from_io` in `read_loop` as well??

Reply:
no it won't